services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/outputs/_watchdog_extensions.py

```diff
@@ -21,6 +24,8 @@
     ],
```

Comment:
TIP: an alternative to ``lambda x, y: x | y`` in ``_EVENTS_TO_WATCH`` you can use ``operator.or_``  https://docs.python.org/3/library/operator.html